gpkit/tests/t_model.py

```diff
@@ -1,16 +1,25 @@
 """Tests for GP and SP classes"""
-import math
 import unittest
+import numpy as np
+from numpy import log
```

Comment:
I think we should either `import np` for `from numpy import (methods)` but not both.